main.cpp

```diff
@@ -332,11 +399,39 @@ struct Dishwasher
         void clogDrain(std::string residues = "too much");
         void tarnish(bool isSensitive = false);
         void shatter(bool isPiledBadly = false);
+        int puttingInGlasses(int toClean = 5, int room = 18)
+        {
+            int spaceLeft = room - glasses - cups;
+            std::cout << "glass(es) to put away: " << toClean << std::endl;
+            while(spaceLeft > 0 && toClean > 0)
+            {
+                //std::cout << toClean << glasses << spaceLeft << std::endl;
+                toClean --;
+                //std::cout << "to clean " << toClean << std::endl;
+                glasses ++;
+                //std::cout << "glasses "  << glasses << std::endl;
+                spaceLeft --;
+                //std::cout << "space left "  << spaceLeft << std::endl;
+            }
+            std::cout << "any glasses left?" << std::endl;
+            toClean == 0 ? std::cout << "all in there" << std::endl : std::cout << "yeah, " << toClean << ", but there's no more room..." << std::endl;
+            return spaceLeft;
+        }
     };
 
     void cleanDishes(Dishes dishes);
     void dryDishes(bool shouldDryDishes = true, int temp = 55);
     void selfClean(int afterRun = 45);
+    int alarm(int time = 100)
+    {
+        time = timePerRun;
+        while(time > 0)
+        {
+            time --;
+            time==0 ? std::cout << std::endl << "beep beep beep" << std::endl : std::cout << "M";
```

Comment:
write this as:
```
std::cout << ( expr ? a : b ) << std::endl; 
```
Don't put the ternary condition first.
also `time` is a reserved keyword, which is why it's blue.
maybe use `currentTime` instead.
- [ ] revise